packages/leptonica.rb

```diff
@@ -4,30 +4,31 @@ class Leptonica < Package
   description 'Software that is broadly useful for image processing and image analysis applications'
   homepage 'http://www.leptonica.com/'
   @_ver = '1.80.0'
-  version @_ver
+  version "#{@_ver}-1"
   license 'Apache-2.0'
   compatibility 'all'
   source_url "https://github.com/DanBloomberg/leptonica/archive/#{@_ver}.tar.gz"
   source_sha256 '3952b974ec057d24267aae48c54bca68ead8275604bf084a73a4b953ff79196e'
 
   binary_url({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/leptonica-1.80.0-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/leptonica-1.80.0-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/leptonica-1.80.0-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/leptonica-1.80.0-chromeos-x86_64.tar.xz'
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/leptonica-1.80.0-1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/leptonica-1.80.0-1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/leptonica-1.80.0-1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/leptonica-1.80.0-1-chromeos-x86_64.tar.xz'
   })
   binary_sha256({
-    aarch64: '92169fedf2fdf6eeecfbcf91046c672827a14293e25eb4529b305fca5e844100',
-     armv7l: '92169fedf2fdf6eeecfbcf91046c672827a14293e25eb4529b305fca5e844100',
-       i686: '0983780d91e159289d9c0efc81390cf7b3b8f666c62848744d7c327005b3d080',
-     x86_64: '9125c52e3bd232009858d406bf37099e0089a543900246f7710b34dbc6b1d7a1'
+    aarch64: 'd4ae243ebd485e3bc061701f07bdd031f02916c79e6e7c0bc4353321c79ec66b',
+     armv7l: 'd4ae243ebd485e3bc061701f07bdd031f02916c79e6e7c0bc4353321c79ec66b',
+       i686: 'b7bc070a1fc98059cdb24339f7bc1dd5bbcacbc0c90a67ec5f35b20f8330a2d2',
+     x86_64: 'b9ab621fe8a76d9b38cc21af0f4ba4f3818b0be7af471baf1995064d20428496'
   })
 
   depends_on 'giflib'
-  depends_on 'libjpeg_turbo'
+  depends_on 'libjpeg'
   depends_on 'libpng'
   depends_on 'libtiff'
   depends_on 'libwebp'
+  depends_on 'openjpeg'
```

Comment:
Do openjpeg and libjpeg have conflicting libraries?

Reply:
No conflict:
```
openjpeg: An open source JPEG 2000 codec, written in C.
/usr/local/bin/opj_compress
/usr/local/bin/opj_decompress
/usr/local/bin/opj_dump
/usr/local/include/openjpeg-2.4/openjpeg.h
/usr/local/include/openjpeg-2.4/opj_config.h
/usr/local/include/openjpeg-2.4/opj_stdint.h
/usr/local/lib64/libopenjp2.a
/usr/local/lib64/libopenjp2.so
/usr/local/lib64/libopenjp2.so.2.4.0
/usr/local/lib64/libopenjp2.so.7
/usr/local/lib64/openjpeg-2.4/OpenJPEGConfig.cmake
/usr/local/lib64/openjpeg-2.4/OpenJPEGTargets.cmake
/usr/local/lib64/openjpeg-2.4/OpenJPEGTargets-release.cmake
/usr/local/lib64/pkgconfig/libopenjp2.pc
```
vs
```
libjpeg: JPEG is a free library for image compression.
/usr/local/bin/cjpeg
/usr/local/bin/djpeg
/usr/local/bin/jpegtran
/usr/local/bin/rdjpgcom
/usr/local/bin/wrjpgcom
/usr/local/include/jconfig.h
/usr/local/include/jerror.h
/usr/local/include/jmorecfg.h
/usr/local/include/jpeglib.h
/usr/local/lib64/libjpeg.a
/usr/local/lib64/libjpeg.la
/usr/local/lib64/libjpeg.so
/usr/local/lib64/libjpeg.so.9
/usr/local/lib64/libjpeg.so.9.4.0
/usr/local/lib64/pkgconfig/libjpeg.pc
/usr/local/share/man/man1/cjpeg.1.gz
/usr/local/share/man/man1/djpeg.1.gz
/usr/local/share/man/man1/jpegtran.1.gz
/usr/local/share/man/man1/rdjpgcom.1.gz
/usr/local/share/man/man1/wrjpgcom.1.gz
```